apps/src/templates/projects/projectsRedux.js

```diff
@@ -82,10 +83,20 @@ function hasOlderProjects(state = initialHasOlderProjects, action) {
   }
 }
 
+const initialPersonalProjectsList = [];
+
+function personalProjectsList(state = initialPersonalProjectsList, action) {
+  return {
+    ...state,
+    projects: action.personalProjectsList,
+  };
```

Comment:
Your reducer function should return `state` unmodified if the action is not `SET_PERSONAL_PROJECTS_LIST`.  As-is, I think you might clear the projects list if any other action runs on the store.